tests/plan.rs

```diff
@@ -7,29 +7,29 @@ const STEAM_DECK: &str = include_str!("./fixtures/linux/steam-deck.json");
 #[cfg(target_os = "macos")]
 const MACOS: &str = include_str!("./fixtures/macos/macos.json");
 
-// Ensure existing plans still parse
-// If this breaks and you need to update the fixture, disable these tests, bump `nix_installer` to a new version, and update the plans.
-#[cfg(target_os = "linux")]
-#[test]
-fn plan_compat_linux() -> eyre::Result<()> {
-    let _: InstallPlan = serde_json::from_str(LINUX)?;
-    Ok(())
-}
+// // Ensure existing plans still parse
```

Comment:
Should these come back, or be dropped forever, or some third option?

Reply:
These should come back!!!!